File: test/react-native/features/fixtures/ios-module/Scenarios/NativeFeatureFlagsScenario.m
#import "NativeFeatureFlagsScenario.h"

@implementation NativeFeatureFlagsScenario

- (void)run: (RCTPromiseResolveBlock)resolve
     reject:(RCTPromiseRejectBlock)reject {

  [Bugsnag clearFeatureFlagWithName:@"should_not_be_reported_1"];
  [Bugsnag clearFeatureFlagWithName:@"should_not_be_reported_2"];
  [Bugsnag clearFeatureFlagWithName:@"should_not_be_reported_3"];

  [Bugsnag addFeatureFlagWithName:@"native_flag"];
}

@end

File: test/react-native/features/fixtures/reactnative/scenarios/NativeFeatureFlagsScenario.kt
package com.reactnative.scenarios

import android.content.Context
import com.bugsnag.android.Bugsnag
import com.facebook.react.bridge.Promise

class NativeFeatureFlagsScenario(context: Context): Scenario(context) {

  override fun run(promise: Promise) {
    Bugsnag.clearFeatureFlag("should_not_be_reported_1")
    Bugsnag.clearFeatureFlag("should_not_be_reported_2")
    Bugsnag.clearFeatureFlag("should_not_be_reported_3")

    Bugsnag.addFeatureFlag("native_flag")
  }
}
